src/app/components/SubPageTitle.tsx

```diff
@@ -23,14 +25,15 @@ const SubPageTitle: FC<SubPageTitleProps> = ({ title, backToLink, style, childre
             spacing={2}
         >
             <Grid item xs sm={4} lg={2} order={1}>
-                <BackButton backToLink={backToLink} />
+            {hiddenBackButton == undefined || hiddenBackButton == false ? <BackButton backToLink={backToLink} /> : null}
```

Comment:
Instead of : null you can use `{(hiddenBackButton == undefined || hiddenBackButton == false) && <BackButton backToLink={backToLink} /> }`